networking/simple-webserver.c

```diff
@@ -0,0 +1,48 @@
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <netinet/in.h>
+#include <string.h>
+
+/*
+  This is a very simple implementation of a web server using C. It is my first contributuion to pdhoot's repository in GitHub.
+  Creates a socket and returns a simple HTML response.
+*/
```

Comment:
Thanks a lot! I will take a look, and make the necessary changes to the code. I did think about creating a while loop there, though just wanted to make it as simple as possible.